### DISCUSSION TITLE
Source/Users/name/.zshenv is it a problem?

### DISCUSSION THREAD

**Discussion by compot13** [Cyfrin/foundry-full-course-cu#1430] [Q&A] [answered]
When I run the command "curl -L https://foundry.paradigm.xyz | bash" at the terminal on VS Code I get this:

Detected your preferred shell is zsh and added foundryup to PATH.
Run 'source /Users/name/.zshenv' or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.

but at the course it's: 

source/Patrick/.bashrc`.

Is it a problem to be ".zshenv" or need to be ".bashrc"

I'm working on MacOS.

**Comment by EngrPips** [comment, marked as answer]
@compot13  Honestly if everything was all set up and you didn't run into any problems, i would say you shouldn't mind that.

**Reply by compot13** [reply]
So far everything is fine, checked it doesn't make any problem

**Reply by EngrPips** [reply]
So you good, enjoy 😉
